### DISCUSSION TITLE
[FEATURE] Display OSM Notes

### DISCUSSION THREAD

**Discussion by gander** [organicmaps/organicmaps#9673] [Ideas]
The only functionality I miss in Organic Maps is the ability to display OSM notes. Not add. Not edit. Not resolve. Display notes. As a default functionality, disabled, to enable. I often use **navigation** and I miss information that there is a note nearby. I can add a GPX, with interesting POIs in the area, e.g. GC, but the notes appear much more dynamically, and importing them every day is a significant inconvenience. It would be much more convenient to simply see them on the map. And then I can handle them in any OSM editor. Such functionality is enough.

**Comment by biodranik** [comment]
Aren't there many other apps already showing OSM notes, including web version of osm.org? Your case is mainly about editing OSM, right? OM allows only a limited number of edits so far.

**Reply by gander** [reply]
Not true. I'm just asking for optional display of notes.

**Reply by gander** [reply]
While navigating, if that wasn't clear.

**Comment by Vuizur** [comment]
I always use Every Door for this, it's a good app.

**Reply by gander** [reply]
Do you use Every Door for navigation? How?

**Comment by jayaddison** [comment]
I would use a feature like this.  I am not an experienced map editor, and I find using a smartphone to edit places fiddly and annoying, so it is not currently something I do while using OrganicMaps.  Instead I prefer to edit later from a computer with a full-size keyboard (e.g. laptop or desktop).

I would like the opportunity to read notes about map regions that I have or am likely to visit -- it can be a good indicator for opportunities to complete surveys and/or to add supporting details.
